Examples/ObjectiveC/AnimatedLineExample.m

```diff
@@ -9,6 +9,7 @@ @interface AnimatedLineExample () <MGLMapViewDelegate>
 @property (nonatomic) MGLShapeSource *polylineSource;
 @property (nonatomic) NSArray<CLLocation *> *locations;
 @property (nonatomic) NSInteger currentIndex;
+@property (nonatomic, strong) NSTimer *timer;
```

Comment:
`strong` is implicit for these property declarations in ARC code, but I suppose being explicit here is also fine.